packages/opencensus-propagation-b3/src/b3-format.ts

```diff
@@ -37,20 +33,12 @@ export class B3Format implements Propagation {
    * @param getter
    */
   extract(getter: HeaderGetter): SpanContext {
-    if (getter) {
-      let opt = getter.getHeader(X_B3_SAMPLED);
-      if (opt instanceof Array) {
-        opt = opt[0];
-      }
-      const spanContext = {
-        traceId: getter.getHeader(X_B3_TRACE_ID),
-        spanId: getter.getHeader(X_B3_SPAN_ID),
-        options: isNaN(Number(opt)) ? undefined : Number(opt)
-      } as SpanContext;
-
-      return spanContext;
-    }
-    return null;
+    const opt = this.parseHeader(getter.getHeader(X_B3_SAMPLED));
+    return {
+      traceId: this.parseHeader(getter.getHeader(X_B3_TRACE_ID)),
```

Comment:
IIUC the `extract` method should [return `null`](https://github.com/census-instrumentation/opencensus-node/blob/3111ab8c29b5d60e68a762c439ef909e3af9589b/packages/opencensus-core/src/trace/propagation/types.ts#L37) if there is no incoming traceID/spanID, rather than fill an object with empty strings.

Reply:
This is valid point, I have changed the implementation to return null in case of no incoming ```traceID/spanID```.